package.json

```diff
@@ -34,7 +34,7 @@
     "docs",
     "package-lock.json"
   ],
-  "main": "./lib/passport-saml",
+  "main": "./lib/passport-saml/index.js",
```

Comment:
I'm OK with this change, and I actually expected this to be this way from the beginning. I guess I'm curious why everything has been working for so long with an abstract `main` (I checked the history) and now it is broken.